### PR TITLE
added get_vsphere_vm function

### PR DESCRIPTION
**Pull request by promiseowolabi**
# get_vsphere_vm

Get summary of vsphere vm's

## Related Issue

https://github.com/rubrikinc/rubrik-sdk-for-python/issues/156

## Motivation and Context

Why is this change required? What problem does it solve?

## How Has This Been Tested?

* Please describe in detail how you tested your changes.
* Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply. If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTION](https://github.com/rubrikinc/rubrik-sdk-for-python/blob/master/CONTRIBUTING.md)** document.
- [ ] I have updated the CHANGELOG file accordingly for the version that this merge modifies.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rubrik_cdm/data_management.py [hunk 1]
```diff
@@ -1620,3 +1620,60 @@ def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
         config = {'vmMoid':moid}
         self.log("vcenter_refresh_vm: Refreshing vSphere VM {} metadata.".format(vm_name))
         self.post('internal', '/vmware/vcenter/{}/refresh_vm'.format(vcenter_id), config, timeout)
+
+    def get_vsphere_vm(self, timeout=15, **query):  # pylint: ignore
```
Comment (drew-russell) [inline rubrik_cdm/data_management.py]: How does `**query` come into play here? Similar to the rest of the Keyword Arguments?

File: rubrik_cdm/data_management.py [hunk 2]
```diff
@@ -1620,3 +1620,60 @@ def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
...
+        """Get summary of all the VMs.
+
+        Keyword Arguments:
+            name {str} -- Search by using a virtual machine name.
```
Comment (drew-russell) [inline rubrik_cdm/data_management.py]: `Keyword Arguments` should have a default value. `Arguments` are the correct way of to describe these (except for `timeout`)

File: rubrik_cdm/data_management.py [hunk 3]
```diff
@@ -1620,3 +1620,60 @@ def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
...
+            offset {int} -- Ignore these many matches in the beginning.
+            moid {str} -- Search by using a virtual machine managed object ID.
+            sla_assignment {str} -- Filter by SLA Domain assignment type. (Direct, Derived, Unassigned)
+            guest_os_name {str} -- Filters by the name of operating system using infix search.
+            sort_by {str} -- Sort results based on the specified attribute. (effectiveSlaDomainName, name, moid, folderPath, infraPath)
+            sort_order {str} -- Sort order, either ascending or descending. (asc, desc)
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            no content.
```
Comment (drew-russell) [inline rubrik_cdm/data_management.py]: Should return the API response

Comment (promiseowolabi) [reply rubrik_cdm/data_management.py]: yes it should, will commit change

File: rubrik_cdm/data_management.py (hunk 1 above)
Comment (promiseowolabi) [reply rubrik_cdm/data_management.py]: Can we use **kwargs to accept keyword arguments and validate the allowed keys within the function? 
Alternative is to define them individually

**Pull request merged**